package.json

```diff
@@ -50,23 +61,37 @@
   ],
   "license": "MIT",
   "dependencies": {
+    "@ljharb/eslint-config": "^12.2.1",
     "async": "^2.6.0",
+    "debug": "^3.1.0",
     "detect-node": "^2.0.3",
     "eslint-config-standard-jsx": "^4.0.2",
     "go-ipfs-dep": "0.4.13",
-    "ipfs-api": "^17.1.3",
+    "guid": "0.0.12",
+    "hapi": "^16.6.2",
```

Comment:
Why not use latest hapi?

Reply:
The latest hapi v17 is a major rewrite for which there is still no updated docs. All the guides/tutorials still reference v16.